test/other/metadce/test_metadce_hello_O0.jssize

```diff
@@ -1 +1 @@
-22913
+23002
```

Comment:
It looks like js size when up in a few of these. Any idea why?

Reply:
These are `-O0` tests in which `ASSERTSIONS` are enabled.  When `ASSERTIONS` are enabled we inject stubs to detect the use of unexported runtime functions.   This means that whenever we add new runtime function the `ASSETIONS` builds get a little bigger.  This is expected and not a problem (IMHO)